src/Raven.Client/Documents/Smuggler/SmugglerResult.cs

```diff
@@ -252,9 +254,6 @@ public override DynamicJsonValue ToJson()
                 if (ClientConfigurationUpdated)
                     json[nameof(ClientConfigurationUpdated)] = ClientConfigurationUpdated;
 
-                if (ConflictSolverConfigUpdated)
```

Comment:
Why this is removed? I see that the value is incorrect

Reply:
I read the value and I assumed is a duplicate of the condition above.
Just an embarrassing mistake.